utils/generator.py

```diff
@@ -119,6 +120,7 @@ def __init__(self):
         # Any interfaces collected here are then rendered as Python in the
         # types.py module.
         self.interfaces = []
+        self.response_interfaces = []
```

Comment:
The main change in the generator is that it now understands if it is generating a type that is used in requests (in other words, a type that users will construct) or in responses (a type that we construct and the user will just access).

For response types we do not generate defaults and dict alternative types.